src/nomad_simulations/outputs.py

```diff
@@ -17,89 +17,214 @@
 #
 
 import numpy as np
-from typing import Optional
+from typing import Any
 from structlog.stdlib import BoundLogger
 
+from nomad.units import ureg
 from nomad.datamodel.data import ArchiveSection
+from nomad.metainfo import (
+    Quantity,
+    SubSection,
+    SectionProxy,
+    Reference,
+    Section,
+    Context,
+    MEnum,
+)
+from nomad.metainfo.metainfo import DirectQuantity, Dimension
 from nomad.datamodel.metainfo.annotations import ELNAnnotation
-from nomad.metainfo import Quantity, SubSection, SectionProxy, Reference
+from nomad.datamodel.metainfo.basesections import Entity
 
-from .atoms_state import AtomsState, OrbitalsState
 from .model_system import ModelSystem
 from .numerical_settings import SelfConsistency
 
 
-class Outputs(ArchiveSection):
-    """
-    Output properties of a simulation. This base class can be used for inheritance in any of the output properties
-    defined in this schema.
+class Variables(ArchiveSection):
+    """ """
 
-    It contains references to the specific sections used to obtain the output properties, as well as
-    information if the output `is_derived` from another output section or directly parsed from the simulation output files.
-    """
+    name = Quantity(type=str)
+    n_bins = Quantity(type=int)
+    bins = Quantity(type=np.float64, shape=['n_bins'])
+    # bins_error = Quantity()
 
-    # TODO add time quantities
+    def normalize(self, archive, logger) -> None:
+        super().normalize(archive, logger)
 
-    normalizer_level = 2
 
-    name = Quantity(
+class Temperatures(Variables):
```

Comment:
Yeah, sounds good. I am thinking on plural because these are an array of temperatures, but it is true that this would be encoded in the plural of `bin`.